tests/test_config_files/switch_grids/new/switch_utm.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "**create_dummy_data": "${config_path}/dummy_data_utm.json"
+  },
+  {
+    "**common_config": "${config_path}/global_config.json",
+    "area": {
+      "manager": "eogrow.core.area.UtmZoneAreaManager",
+      "area_filename": "test_area.geojson",
+      "area_buffer": 0.01,
+      "patch_size_x": 3600,
+      "patch_size_y": 3000,
+      "patch_buffer_x": 120,
+      "patch_buffer_y": 50,
+      "offset_x": 100,
+      "offset_y": 20
```

Comment:
do these offsets not affect the split? I guess not, because the splitter is working directly with produced geometries, but just wanted to make sure.

Reply:
no, it works entirely on geometries
i wanted to leave them in to ensure it doesnt inerfere